CreateRoom/Program.cs

```diff
@@ -0,0 +1,104 @@
+            updateRoomParticipants.Add(updateParticipant1);
+            updateRoomParticipants.Add(updateParticipant2);
+
+            Response<RoomModel> updateRoomResponse = await roomsClient.UpdateRoomAsync(roomId, updateValidFrom, updateValidUntil, null, updateRoomParticipants);
+            RoomModel updatedCommunicationRoom = updateRoomResponse.Value;
+
+
+            // Get Room
+            Response<RoomModel> getRoomResponse = await roomsClient.GetRoomAsync(roomId);
+            RoomModel room = getRoomResponse.Value;
+
+            // Add Room Participants
+            var communicationUser6 = "<CommunicationUserIdentifier.Id>";
+            var communicationUser7 = "<CommunicationUserIdentifier.Id>";
+            List<RoomParticipant> addRoomParticipants = new List<RoomParticipant>();
+            RoomParticipant addParticipant1 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser6), RoleType.Attendee);
+            RoomParticipant addParticipant2 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser7), RoleType.Attendee);
+            addRoomParticipants.Add(addParticipant1);
+            addRoomParticipants.Add(addParticipant2);
+            Response<ParticipantsCollection> addParticipantsResponse = await roomsClient.AddParticipantsAsync(roomId, addRoomParticipants);
+            ParticipantsCollection roomParticipantsAfterAddingParticipants = addParticipantsResponse.Value;
+
+            // Update Room Participants
+            List<RoomParticipant> UpdateRoomParticipants = new List<RoomParticipant>();
+
+            RoomParticipant updateRoomParticipant1 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser6), RoleType.Presenter);
+            RoomParticipant updateRoomParticipant2 = new RoomParticipant(new CommunicationUserIdentifier(communicationUser7), RoleType.Consumer);
+            UpdateRoomParticipants.Add(updateRoomParticipant1);
+            UpdateRoomParticipants.Add(updateRoomParticipant2);
+            Response<ParticipantsCollection> updateParticipantsResponse = await roomsClient.UpdateParticipantsAsync(roomId, updateRoomParticipants);
+            ParticipantsCollection roomParticipantsAfterUpdatingParticipants = updateParticipantsResponse.Value;
+
+            // Update Room Participants
+            List<CommunicationIdentifier> RemoveRoomParticipants = new List<CommunicationIdentifier>();
+            RemoveRoomParticipants.Add(new CommunicationUserIdentifier(communicationUser6));
+            Response<ParticipantsCollection> removeParticipantsResponse = await roomsClient.RemoveParticipantsAsync(roomId, RemoveRoomParticipants);
+            ParticipantsCollection roomParticipantsAfterRemovingParticipants = removeParticipantsResponse.Value;
+
+            // Get Room Participants
+            RemoveRoomParticipants.Add(new CommunicationUserIdentifier(communicationUser6));
```

Comment:
This line does not appear to be required, please consider removing it

Reply:
Addressed. Thanks.